dev-qt/qt-creator/qt-creator-9999.ebuild

```diff
@@ -103,7 +103,20 @@ unset x
 PATCHES=( "${FILESDIR}"/${PN}-4.12.0-dylib-fix.patch )
 
 pkg_setup() {
-	use clang && llvm_pkg_setup
+	if use clang; then
+		llvm_pkg_setup
+		local llvm_prefix=$(get_llvm_prefix "${LLVM_MAX_SLOT}")
+		local format_file="${llvm_prefix}"/include/clang/Format/Format.h
+		local patched_string="^#define KEEP_LINE_BREAKS_FOR_NON_EMPTY_LINES_BACKPORTED$"
+		if ! grep "${patched_string}" "${format_file}" &> /dev/null; then
```

Comment:
This code would check header in only one slot (`sys-devel/clang:10`), but dependencies in this ebuild claim to support 3 slots (`sys-devel/clang:10`, `sys-devel/clang:9`, `sys-devel/clang:8`).

Reply:
Yes.  It can potentially build against multiple clang slots but builds against only one.  `src_configure` passes `LLVM_INSTALL_DIR="$(get_llvm_prefix ${LLVM_MAX_SLOT})")` to qmake so that would be the correct header.